sklearn/tests/test_naive_bayes.py

```diff
@@ -228,16 +262,18 @@ def test_discretenb_pickle(cls):
 
     assert_array_equal(y_pred, clf.predict(X2))
 
-    # Test pickling of estimator trained with partial_fit
-    clf2 = cls().partial_fit(X2[:3], y2[:3], classes=np.unique(y2))
-    clf2.partial_fit(X2[3:], y2[3:])
-    store = BytesIO()
-    pickle.dump(clf2, store)
-    clf2 = pickle.load(BytesIO(store.getvalue()))
-    assert_array_equal(y_pred, clf2.predict(X2))
+    if cls is not CategoricalNB:
```

Comment:
what's happening here?

Reply:
I forgot to change it back when I reimplemented `partial_fit`.